app/src/main/res/values/dimen.xml

```diff
@@ -4,4 +4,16 @@
     <dimen name="date_home">14sp</dimen>
     <dimen name="big_text_home">20sp</dimen>
     <dimen name="small_text_home">14sp</dimen>
-</resources>
\ No newline at end of file
+
+    <!-- ImageView for error_image -->
+    <dimen name="layout_marginTop">150dp</dimen>
+    <dimen name="padding_start">60dp</dimen>
+    <dimen name="padding_end">20dp</dimen>
+
+    <!-- TextView for error_message -->
+    <dimen name="marginTop">20dp</dimen>
+    <dimen name="textSize">17sp</dimen>
+
+    <!-- TextView for retry_button -->
+    <dimen name="textMarginTop">20dp</dimen>
+</resources>
```

Comment:
If we have three dimensions at 20dp, why do we merge them into one single dimension named `margin_20`